src/RPC.ts

```diff
@@ -0,0 +1,17 @@
+import { ONCHAIN_KIT_CONFIG } from './OnchainKitConfig';
+
+/**
+ * Access the RPC URL for OnchainKit.
+ * Defaults to using Coinbase Developer Platform if a RPC URL is not provided.
+ */
+export const getRPCUrl = () => {
```

Comment:
the file should be named `getRPCUrl.ts`, as in this way it's explicit what it does. Have it call RPC just creates confusion.